test/build-options.js

```diff
@@ -0,0 +1,79 @@
+import {expect} from 'chai';
+import buildOptions from '../src/build-options';
+
+describe('buildOptions', () => {
+
+  it('takes up to two sets of options and merges them', () => {
+    const result = buildOptions({ foo: 'bar' }, { fizz: 'buzz' });
+
+    expect(result).to.deep.equal({ foo: 'bar', fizz: 'buzz' });
+  });
+
+  it('doesnt have to be given anything', () => {
```

Comment:
This test name doesn't exactly describe what is under review. There is an assertion being made on the response.